src/it/scala/ApiServiceISpec.scala

```diff
@@ -0,0 +1,146 @@
+import cats.effect.IO
+import cats.effect.kernel.Resource
+import cats.effect.testing.scalatest.AsyncIOSpec
+import jms4s.JmsAutoAcknowledgerConsumer.AutoAckAction
+import jms4s.config.QueueName
+import jms4s.jms.JmsMessage
+import jms4s.sqs.simpleQueueService
+import jms4s.sqs.simpleQueueService.{ Config, Credentials, DirectAddress, HTTP }
+import org.scalatest.{ BeforeAndAfterAll, BeforeAndAfterEach }
+import org.scalatest.concurrent.{ Eventually, IntegrationPatience }
+import org.scalatest.freespec.AsyncFreeSpec
+import org.scalatest.matchers.must.Matchers
+import org.typelevel.log4cats.slf4j.Slf4jFactory
+import org.typelevel.log4cats.{ LoggerFactory, SelfAwareStructuredLogger }
+import uk.gov.nationalarchives.omega.api.conf.ServiceConfig
+import uk.gov.nationalarchives.omega.api.services.ApiService
+
+import java.nio.file.{ Files, Paths }
+import javax.jms.{ Connection, MessageProducer, Session }
+import scala.concurrent.duration.DurationInt
+import scala.io.Source
+import scala.util.{ Failure, Success, Using }
+
+class ApiServiceISpec
+    extends AsyncFreeSpec with AsyncIOSpec with Matchers with Eventually with IntegrationPatience
+    with BeforeAndAfterEach with BeforeAndAfterAll {
+
+  implicit val loggerFactory: LoggerFactory[IO] = Slf4jFactory[IO]
+  implicit val logger: SelfAwareStructuredLogger[IO] = LoggerFactory[IO].getLogger
+
+  private val apiService = new ApiService(
```

Comment:
We could add constants for both queue names.

Reply:
Best to add `temp` to `.gitignore`, and maybe call it `temp-integration-tests` - or better yet, remove that directory in the `afterAll`? I see that we're deleting the reply messages but not the requests.

Reply:
I added constants for the queue names.

The temporary files should no longer be an issue after completion of [PACT-811](https://national-archives.atlassian.net/browse/PACT-811).

Reply:
Confirmed (both).